### ISSUE TITLE
Apps and pages do not fully load

### ISSUE THREAD

**Issue by Tazovsky** [wavebox/waveboxapp#1337]
After I restarted my Mac M1 (macOS 11.2.3 (20D91)) apps and pages are not being fully loaded:

![image](https://user-images.githubusercontent.com/29068994/118096621-e1a29f00-b3d1-11eb-87d7-6f0c6c5a23cf.png)
![image](https://user-images.githubusercontent.com/29068994/118096637-e8311680-b3d1-11eb-81c1-99259bedea0a.png)
![image](https://user-images.githubusercontent.com/29068994/118096658-f121e800-b3d1-11eb-9017-1655cecea232.png)

I even reinstalled Wavebox, but it didn't help.

**Comment by Thomas101** [comment]
Are you using the apple silicon edition of Wavebox from https://wavebox.io/download or the intel version? We recommend using the apple silicon one so it doesn't run under Rosetta.

Disabling hardware acceleration might help. You can do this a couple of ways...

- Open Wavebox 'Settings > Advanced > Disable hardware acceleration' >  Toggle OFF
- Paste 'chrome://settings' into any Wavebox address bar and then search for 'hardware acceleration' and disable it.
- Launch Wavebox using the terminal app by typing:
/Applications/Wavebox.app/Contents/MacOS/Wavebox --disable-gpu
​
If disabling hardware acceleration does work, then you can enable this setting permanently by doing 1 or 2

**Comment by Tazovsky** [comment]
@Thomas101 it worked. Thanks. I use Apple Silicon version, Do you have any idea why it broke after restarting the app? And are you working on fix?

**Comment by Thomas101** [comment]
This is the first we've seen of this. Did disabling hardware acceleration make any difference?

Can you reach out to support (support@wavebox.io) we can generate some diagnostics and see if we can get to the bottom of it

**Comment by Tazovsky** [comment]
> Did disabling hardware acceleration make any difference?

@Thomas101 yes, it works as expected after disabling hardware acceleration. I also sent email to support@wavebox.io.
